tests/linearizability/failpoints.go

```diff
@@ -165,6 +171,42 @@ func (f goPanicFailpoint) pickMember(t *testing.T, clus *e2e.EtcdProcessCluster)
 	}
 }
 
+func (f goPanicFailpoint) Available(member e2e.EtcdProcess) bool {
+	fs, err := listFailpoints(member)
+	if err != nil {
+		return false
+	}
+	return f.available(fs)
+}
+
+func (f goPanicFailpoint) available(fs []string) bool {
+	for _, failpoint := range fs {
+		if f.failpoint == failpoint {
+			return true
+		}
+	}
+	return false
+}
+
+func listFailpoints(member e2e.EtcdProcess) ([]string, error) {
```

Comment:
shell it return map[string]struct{} -> so effectively a set for easier presence checking ?